packages/twenty-server/src/engine/middlewares/utils/bind-data-to-request-object.utils.ts

```diff
@@ -0,0 +1,16 @@
+import { Request } from 'express';
+
+import { AuthContext } from 'src/engine/core-modules/auth/types/auth-context.type';
+
+export const bindDataToRequestObject =
+  (data: AuthContext) =>
+  (request: Request, metadataVersion: number | undefined) => {
```

Comment:
why creating a callback? can we just have a 3 parameter function here?
```suggestion
  (data: AuthContext, request: Request, metadataVersion: number | undefined) => {
```

Reply:
Makes sense, wanted to separate the payload from the context, but I am updating it